platform-sdk/swirlds-merkledb/src/main/java/com/swirlds/merkledb/MerkleDbDataSource.java

```diff
@@ -170,6 +171,7 @@ public final class MerkleDbDataSource implements VirtualDataSource {
 
     public MerkleDbDataSource(
```

Comment:
Most changes in MerkleDbDataSource constructor are just to move code up and down. This is needed to make sure path to KV store is initialized before key to path map (currently it is not).